app/services/onetime/backfill_purchase_refund_policy_max_refund_period.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Onetime::BackfillPurchaseRefundPolicyMaxRefundPeriod < Onetime::Base
```

Comment:
```suggestion
class Onetime::SetMaxAllowedRefundPeriodForPurchaseRefundPolicies < Onetime::Base
```

Use `Onetime::SetMaxAllowedRefundPeriodForProductRefundPolicies` for inspiration.